HOWTO-port-an-exercise.md

```diff
@@ -15,36 +15,57 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
     - add the labels: `x:action/create`, `x:module/practice-exercise`,
       `x:status/claimed`, `x:type/content`
     - assign it to yourself.
-1. copy the exercises/practice/foo-bar directory from the bash repo.
-1. in the exercises/practice/foo-bar directory, **delete** `foo_bar.sh`, and
-   `.meta/{test.toml,version,example.sh}`
-1. rename `foo_bar.bats` to `test-foo-bar.bats`
-    - note the underscore changed: the AWK track uses kebab case
-1. edit `.meta/config.js`
-    - fix the filenames from .sh to .awk, and the test file
-    - set yourself as the author, and empty the contributors list.
-1. edit `test-foo-bar.awk` so the tests are AWK specific. 
-    - see other exercises for the track style.
-    - remember that the tests are the requirements that students use to
-      create their solutions: the tests should strive to be straightforward
-      and self-explantory.
-1. create `foo-bar.awk` so all the tests pass.
-1. move the solution to `.meta/example.awk`
-1. copy `../hamming/hamming.awk` to `foo-bar.awk` -- this is the stub file
-   that students first see.
-1. cd to the track root and:
-    ```sh
-    bin/fetch-configlet
-    bin/configlet fmt -u -e foo-bar
-    bin/configlet sync -u -e foo-bar
-    bin/configlet lint
-    ```
-    And make sure those all pass.
-1. commit your code and run
-    ```sh
-    bin/validate_one_exercise exercises/practice/foo-bar
-    ```
-    And make sure it passes
+
```

Comment:
This works. I'm glad you kept the original english steps as somments.

If we add another bullet point here:
```none
1. follow along with these steps to create the exercise:
```
and then indent the code block by 4 spaces, then the "push your branch" item at the bottom will render as "3. ..."